app/src/main/res/values/strings.xml

```diff
@@ -68,6 +69,83 @@
         <item>CH2</item>
     </string-array>
 
+    <string-array name="multimeter_knob_states_title">
+        <item>Channel 1</item>
+        <item>CAP</item>
+        <item>Voltage</item>
+        <item>Resistance</item>
+        <item>Capacitance</item>
+        <item>Logic Analyzer Pin 1</item>
+        <item>Logic Analyzer Pin 2</item>
+        <item>Logic Analyzer Pin 3</item>
+        <item>Logic Analyzer Pin 4</item>
+        <item>Channel 2</item>
+        <item>Channel 3</item>
+    </string-array>
```

Comment:
Where's the entry for `MIC` pin?

Reply:
I referred to knob states of multimeter, and there wasn't an entry for MIC. Should I add mic here and on multimeter as well?